core/src/main/resources/reference.conf

```diff
@@ -563,10 +579,13 @@ load-control {
   key-value-read = 10000
   key-value-write = 10000
   # The I/O queue has the specificity to be bounded. This sets its size
-  io-queue-size = 10000
+  io-queue-size = 500
   # If the I/O queue is full, subsequent requests are rejected and to be retried later.
-  # This time window specifies how much time without request rejection consititutes a "back to normal load" event
-  io-normal-window = 10000
+  # This time window specifies how much time without request rejection constitutes a "back to normal load" event.
+  # The amount of time to be waited without request rejection can vary between `io-normal-window-minimum` and
+  # `io-normal-window-maximum` depending on severity (how long I/O commands have been waiting to run).
+  io-normal-window-minimum = 20 seconds
+  io-normal-window-maximum = 60 seconds
```

Comment:
Changed config parameters, this allows for scaling the amount of backpressure to the severity of the backup.